docs/source/manim_directive.py

```diff
@@ -147,6 +155,7 @@ def run(self):
         media_dir = os.path.join("source", "media")
         images_dir = os.path.join(media_dir, "images")
         video_dir = os.path.join(media_dir, "videos")
+        output_file = f"{clsname}-{classnamedict[clsname]}"
```

Comment:
Maybe we can set here something more unambiguous like :
output_file = f"{clsname}-occurrence{classnamedict[clsname]}"

Reply:
Sure, any modification will work.

Given that nobody is ever supposed to interact with these files (except for the docbuild system, which is pretty verbose about what it is currenly processing anyways), I went with a very minimal version. But I'm happy to be more explicit. :-)